Source/JavaScriptCore/bytecode/StructureStubInfo.h

```diff
@@ -473,6 +474,7 @@ inline auto appropriateOptimizingGetByIdFunction(AccessType type) -> decltype(&o
     case AccessType::GetByIdDirect:
         return operationGetByIdDirectOptimize;
     case AccessType::GetPrivateName:
+    case AccessType::GetPrivateNameById:
```

Comment:
Add handling of this in appropriateGenericGetByIdFunction too and drop `AccessType::GetPrivateName` from `appropriateGenericGetByIdFunction`.

Reply:
Added call to `operationGetPrivateNameByIdGeneric` in `appropriateGenericGetByIdFunction`.
Removed `AccessType::GetPrivateName` case